cocos/particle/particle-system-component.ts

```diff
@@ -61,157 +69,198 @@ export class ParticleSystemComponent extends RenderableComponent {
+     * @zh 是否开启粒子分别在 XYZ 轴的旋转角度
+     * @property {boolean} startRotation3D
+     */
     @property({
         displayOrder: 12,
-        tooltip:'粒子初始旋转角度',
+        tooltip:'i18n:particle.start_rotation_3d',
     })
     public startRotation3D = false;
 
     /**
-     * @zh 粒子初始旋转角度。
+     * @en Particle initial rotation angle
+     * @zh 粒子初始旋转角度
+     * @property {CurveRange} startRotationX
      */
     @property({
         type: CurveRange,
         range: [-1, 1],
         radian: true,
         displayOrder: 12,
-        tooltip:'粒子初始旋转角度'
+        tooltip:'i18n:particle.start_rotation',
     })
     public startRotationX = new CurveRange();
 
     /**
-     * @zh 粒子初始旋转角度。
+     * @en Particle initial rotation angle
+     * @zh 粒子初始旋转角度
+     * @property {CurveRange} startRotationY
      */
     @property({
         type: CurveRange,
         range: [-1, 1],
         radian: true,
         displayOrder: 12,
-        tooltip:'粒子初始旋转角度'
+        tooltip:'i18n:particle.start_rotation',
     })
     public startRotationY = new CurveRange();
 
     /**
-     * @zh 粒子初始旋转角度。
+     * @en Particle initial rotation angle
+     * @zh 粒子初始旋转角度
+     * @property {CurveRange} startRotationZ
      */
     @property({
         type: CurveRange,
         range: [-1, 1],
         radian: true,
         displayOrder: 12,
         formerlySerializedAs: 'startRotation',
-        tooltip:'粒子初始旋转角度'
+        tooltip:'i18n:particle.start_rotation',
     })
     public startRotationZ = new CurveRange();
 
     /**
+     * @en Delay particle emission time after particle system starts running.
      * @zh 粒子系统开始运行后，延迟粒子发射的时间。
+     * @property {CurveRange} startDelay
      */
     @property({
         type: CurveRange,
         displayOrder: 6,
-        tooltip:'粒子系统开始运行后，延迟粒子发射的时间',
+        tooltip:'i18n:particle.start_delay',
     })
     public startDelay = new CurveRange();
 
     /**
+     * @en Particle life cycle。
      * @zh 粒子生命周期。
+     * @property {CurveRange} startLifetime
      */
     @property({
         type: CurveRange,
         displayOrder: 7,
-        tooltip:'粒子生命周期',
+        tooltip:'i18n:particle.start_life_time',
     })
     public startLifetime = new CurveRange();
 
     /**
-     * @zh 粒子系统运行时间。
+     * @en The run time of particle.
+     * @zh 粒子系统运行时间
+     * @property {Number} duration
      */
     @property({
         displayOrder: 0,
-        tooltip:'粒子系统运行时间',
+        tooltip:'i18n:particle.duration',
     })
     public duration = 5.0;
 
     /**
-     * @zh 粒子系统是否循环播放。
+     * @en Whether the particle system loops.
+     * @zh 粒子系统是否循环播放
+     * @property {Boolean} loop
      */
     @property({
         displayOrder: 2,
-        tooltip:'粒子系统是否循环播放',
+        tooltip:'i18n:particle.loop',
     })
     public loop = true;
 
     /**
-     * @zh 选中之后，粒子系统会以已播放完一轮之后的状态开始播放（仅当循环播放启用时有效）。
+     * @en When selected, the particle system will start playing after one round has been played (only effective when loop is enabled).
```

Comment:
When selected, the particle system will start playing in the state as if it has been played one life cycle (only effective when loop is enabled).